docs/reference/supported-environments.md

```diff
@@ -50,35 +68,58 @@ Requirements for the components can be seen below:
 Camunda 8 works with the [default distribution](https://www.elastic.co/downloads/elasticsearch) of Elasticsearch, which is available under the [Free or Gold+ Elastic license](https://www.elastic.co/pricing/faq/licensing#summary).
 :::
 
+### Volume Performance
+
+Depending on your cloud provider, we offer the following volume performance recommendations:
```

Comment:
Also here, can we be specific please (recommendation vs minium requirement)? I would love the see minium requirements.

Reply:
We should define volume performance parameters to cover on-premises or "exotic cloud"  (i.e. Oracle, IBM, Digital Ocean, hosting providers)

Reply:
We come across questions about the support of NFS. In some environments, NFS could perform as well as local storage, and we need to define required performance characteristics and maybe call out some NFS's that we know are not supported.

Reply:
@theburi I view both of these items as feature enhancements and outside of the scope of this PR. Can you create epics to drive these changes with the appropriate team (Distribution or InfraEx)?

Reply:
Yes. Let's add this reference.